js/core/src/init.svelte.ts

```diff
@@ -4,7 +4,8 @@ import {
 	get_inputs_outputs
 } from "./init_utils";
 import { translate_if_needed } from "./i18n";
-import { tick } from "svelte";
+import { tick, getContext } from "svelte";
```

Comment:
The main changes are in this file:
- `untrack_children_of_closed_accordions_or_inactive_tabs` sets visibility=False for children of unopened accordions, inactive tabs
- changes to `update_state`. If a component with `visible=False` is the output of an event, we would previously not update its props. Now we do. We also let components with `visible=False`, trigger change events. This was true in 5.x. In order to trigger change events from `update_state`, we have to pass in the `gradio_dispatch_function` from blocks
- Last big change is that tabs/accordions now dispatch custom events that let the AppTree know when to render the children of an Accordion/Tabs. You can see that in `js/tabs/`, `js/accodion`, and `Blocks.svelte` mostly
-